fp.js

```diff
@@ -6,7 +6,7 @@ var people = [
   {name: "Jessica", age: 34},
   {name: "Jasmine", age: 39}
 ]
-people.forEach(function(person){
+people.map(function(person){
   peoplesAges.push(person.age)
```

Comment:
You don't need to push anything here. Instead, you can (1) save the result of `.map` directly to a variable and (2) make sure to return exactly what you want to see in the resulting array. So the solution would look like this...

```js
let ages = people.map(function(person){
  return person.age
})
```